src/js/components/Pagination/Pagination.js

```diff
@@ -199,27 +204,72 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    const paginationControls = (
+      <Nav
+        a11yTitle={ariaLabel || a11yTitle || 'Pagination Navigation'}
+        ref={ref}
+      >
+        <Box as="ul" {...theme.pagination.controls}>
+          {controls.map((control, index) => (
+            /* Using index as key (as opposed to a unique id) seems to
+             * help React prioritize rendering the updated controls as
+             * desired. Whereas, using a unique id resulted in rendering
+             * the active control with an undesired lag. */
+            // eslint-disable-next-line react/no-array-index-key
+            <PageControl key={index} size={size} {...control} />
+          ))}
+        </Box>
+      </Nav>
+    );
+
+    // for backwards compatibility
+    if (!summary && !stepOptions)
+      return (
+        <StyledPaginationContainer
+          flex={false}
+          {...theme.pagination.container}
+          {...rest}
+        >
+          {paginationControls}
+        </StyledPaginationContainer>
+      );
+
     return (
       <StyledPaginationContainer
+        direction="row"
+        align="center"
+        gap={{ column: 'medium', row: 'xsmall' }}
```

Comment:
Is it intentional that this row gap is `xsmall` but the row gap between stepselector and pagination controls is `small`?

Just double checking

Reply:
let me double check!